scripts/nightly/upload.py

```diff
@@ -28,6 +36,7 @@
     Body=json.dumps(
         {
             "url": "https://releases.wagtail.org/nightly/dist/" + f.name,
+            "latestUrl": "https://releases.wagtail.org/nightly/dist/latest.whl",
```

Comment:
I'm not sure if this is a good idea, I guess this could cause code that uses `grep` or a similar tool that looks for `https://[^"]*`, such as how I did it for [our bakerydemo nightly](https://github.com/wagtail/bakerydemo/blob/6a16f1b7d3c53d424361825abb9507032b5b31a4/Dockerfile#L38-L40), to end up with multiple results.